.github/workflows/ci.yml

```diff
@@ -19,6 +19,8 @@ jobs:
           java-version: ${{ matrix.javaversion }}
       - name: Build with Maven
         run: mvn --batch-mode install -Dgpg.skip=true -DskipTests=true -Dcheckstyle.skip=true
+      - name: Test with Maven
+        run: mvn test -B
```

Comment:
```suggestion
        run: mvn --batch-mode install -Dgpg.skip=true -Dcheckstyle.skip=true
```

Reply:
Instead of having an explicit step, we can simply remove the flag that skips tests as a part of the build process since by default in Java it will run both.